tests/integration/test.replication.js

```diff
@@ -57,14 +57,29 @@ adapters.forEach(function (adapters) {
         });
       }
 
+      if (testUtils.isSyncGateway()) {
+        changes = changes.filter(function(change){
+          return change.id !== "_user/";
+        });
+      }
+
       return changes;
     }
 
     function verifyInfo(info, expected) {
       if (!testUtils.isCouchMaster()) {
-        info.update_seq.should.equal(expected.update_seq, 'update_seq');
+        if (typeof info.doc_count === 'undefined') { 
+          // info is from Sync Gateway, which allocates an extra seqnum
+          // for user access control purposes.
```

Comment:
^ let's use `isSyncGateway()` then to be clearer. `should.be.within` is a fine test.